core/src/main/java/io/questdb/griffin/SqlOptimiser.java

```diff
@@ -3851,9 +3857,6 @@ private void rewriteCountDistinct(QueryModel model) throws SqlException {
             countDistinctExpr.token = "count";
             countDistinctExpr.paramCount = 0;
             countDistinctExpr.rhs = null;
-
```

Comment:
This code is changed by #4710. Would be awesome if you could review that PR, so that we can merge it.